### ISSUE TITLE
JAWS does not announce a <td> element with "contenteditable" attribute as editable

### ISSUE THREAD

**Issue by sarasuri** [FreedomScientific/VFO-standards-support#859]
### Summary

JAWS does not identify the `<td>` element with the "contenteditable" attribute as editable. Instead, it only speakes the content within the table cell without indicating that it is editable. However, JAWS properly recognizes and announces the "contenteditable" attribute when applied to "`<p>`" or "`<div>`" element.

### Expected result

JAWS should announce a table cell "`<td>`" with the "contenteditable" attribute as editable. 

### Actual result

JAWS only reads the content within the table cell and does not indicate that it is editable.

### Example

 ```
<table width="100%" border="1">
  <tr>
    <th scope="col">Test 1</th>
    <th scope="col">Test 2</th>
    </tr>
  <tr>
  <td contenteditable="true">134</td>
    <td contenteditable="true">542</td>
  </tr>
</table>
```

#### JAWS version and build number
JAWS version 2020.2008.24

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome Version 127.0.6533.89 (Official Build) (64-bit)
Firefox version 129.0
Microsoft Edge Version 127.0.2651.86 (Official build) (64-bit)

**Comment by sarasuri** [comment]
Just checking if there is any update or feedback regarding this issue.
Thank you

**Comment by stevefaulkner** [comment]
test case [content editable `td`](https://codepen.io/stevef/pen/BagPQgz)

**Comment by stevefaulkner** [comment]
triaged, no announcement in JAWS 2024.2406.121 chrome Version 127.0.6533.101

NVDA announces as multiline edit when focused

**Comment by stevefaulkner** [comment]
The `<td>` is exposed via acc API as editable in chrome, Same as a `<p>` is, via `editable` property with a value of `richtext`

But JAWS does not indicate `<td>` is editable. It would also be useful to indicate it is _multiline_ as NVDA does
